test/unit/test_quantum_computer.py

```diff
@@ -594,3 +599,221 @@ def check_random_columns(oa, strength):
             oa = _construct_orthogonal_array(num_q, strength=strength)
             for _ in range(10):
                 check_random_columns(oa, strength)
+
+
+def test_qc_expectation(client_configuration: QCSClientConfiguration, dummy_compiler: DummyCompiler):
```

Comment:
There were two `test_quantum_computer.py` files. These tests are not newly-written, just appended here from that other file.